deepchem/feat/base_classes.py

```diff
@@ -421,3 +421,45 @@ class UserDefinedFeaturizer(Featurizer):
   def __init__(self, feature_fields):
     """Creates user-defined-featurizer."""
     self.feature_fields = feature_fields
+
+
+class DummyFeaturizer(Featurizer):
+  """Class that implements a no-op featurization.
+  This is useful when the raw dataset has to be used without
+  featurizing the examples.
```

Comment:
Might want to add here that it's particularly useful in pipelines like loading a MolNet dataset where a featurizer is required.